flang/lib/Support/Fortran-features.cpp

```diff
@@ -8,11 +8,75 @@
 
 #include "flang/Support/Fortran-features.h"
 #include "flang/Common/idioms.h"
+#include "flang/Parser/characters.h"
 #include "flang/Support/Fortran.h"
+#include <string>
+#include <string_view>
 
 namespace Fortran::common {
 
+// Namespace for helper functions for parsing CLI options
+// used instead of static so that there can be unit tests for these
+// functions.
+namespace featuresHelpers {
+
+static std::vector<std::string_view> SplitCamelCase(std::string_view x) {
```

Comment:
This function is static, so it doesn't need to be in a namespace.  The following function can probably also be static, and then you don't need the namespace.